### DISCUSSION TITLE
Windows 11 quick launch toolbar icons are too small.

### DISCUSSION THREAD

**Discussion by robtgmann** [valinet/ExplorerPatcher#2595] [Q&A]
large icons are set and show up in the main task bar, but icons in the quick launch toolbar are almost too small to be useful -- even though displayed on a 32 inch monitor.  Help please.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Reply by robtgmann** [reply]
This problem is occurring on a new LG laptop, model 17z90R with 32GB of Ram.  
OS is Windows Pro, Version 23H2, OS build 22631.2861  
Explorerpatcher build is 22621.2861.62.1  (updated 12/17/23 at about 5:30 pm Pacific Time)

Sorry to say I do not remember how I arrived at this configuration.  I believe I started by working with StartAllBack, alternating and finally settling on ExplorerPatcher. StartAllBack was completely removed from the system, and is no longer present.  (used Revo Uninstaller Pro).

I can't, for instance remember how I installed the quick launch tool bar, or how I found the setting that let me choose 'small icons."  I do remember that when I was dissapointed with the icon size I tried the small icon option and it was smaller still.  Attached is a partial screen shot of my quick launch tool bar and, for comparison a single icon in the regular task bar.  On my 32 inch monitor the QL icons measure 4+ mm in diameter!.

![Icons 2023-12-17 122747](https://github.com/valinet/ExplorerPatcher/assets/88255837/c29ed74b-f440-48fe-98f6-8160f15e5a2f)

As to which settings I have changed in ExplorerPatcher, of course I don't remember.  Here are some settings that may be germane:
Taskbar style is Windows 10
Automatically hide the TB is off
Start button style Windows 10
All others on the TB page are default except Combine is combine when full.

On the FE page there are 10 options to change:  I have checked numbers 1 ,2, 5, 8.  The control interface is Windows 10 ribon.

On the Start menu 
Style is Windows 20
Position is at edge
Frequent apps is 6
Multiple displays is default (I run a 32 Samsung and a 24 Dell monitor.)
Show tiles is checked
Corner, not rounded
Display mode default
App list Display.

Windows switcher is Windows 11 (default)

Weather is X

Other -- there are 13 oppportunities for change.  I have checked numbers 2, 3, 4, 7, 11, 12, 13. Disabled sleep is default.

Updates:  The two options are not checked

Advanced:  There are 11 oporutnities for change.  I have checked numbers 8 and 10. 10 is set to None (default)

Please let me know if you need other information.  I apprecreiate you taking the time to look at this.

**Comment by robtgmann** [comment]
I’ve posted a reply on Git Hub.  If you need a direct version, here is a copy:
 
 
This problem is occurring on a new LG laptop, model 17z90R with 32GB of Ram.  
OS is Windows Pro, Version 23H2, OS build 22631.2861  
ExplorerPatcher build is 22621.2861.62.1  (updated 12/17/23 at about 5:30 pm Pacific Time)
 
 
Sorry to say I do not remember how I arrived at this configuration.  I believe I started by working with StartAllBack, alternating and finally settling on ExplorerPatcher. StartAllBack was completely removed from the system, and is no longer present.  (used Revo Uninstaller Pro).
 
 
I can't, for instance remember how I installed the quick launch tool bar, or how I found the setting that let me choose 'small icons."  I do remember that when I was disappointed with the icon size I tried the small icon option and it was smaller still.  Attached is a partial screen shot of my quick launch tool bar and, for comparison a single icon in the regular task bar.  On my 32 inch monitor the QL icons measure 4+ mm in diameter!.
 

	
		

 
As to which settings I have changed in ExplorerPatcher, of course I don't remember.  Here are some settings that may be germane:
Taskbar style is Windows 10
Automatically hide the TB is off
Start button style Windows 10
All others on the TB page are default except Combine is combine when full.
 
On the FE page there are 10 options to change:  I have checked numbers 1 ,2, 5, 8.  The control interface is Windows 10 ribbon.
 
On the Start menu 
Style is Windows 20
Position is at edge
Frequent apps is 6
Multiple displays is default (I run a 32 Samsung and a 24 Dell monitor.)
Show tiles is checked
Corner, not rounded
Display mode default
App list Display.
 
Windows switcher is Windows 11 (default)
 
Weather is X
 
Other -- there are 13 opportunities for change.  I have checked numbers 2, 3, 4, 7, 11, 12, 13. Disabled sleep is default.
 
Updates:  The two options are not checked
 
Advanced:  There are 11 opportunities for change.  I have checked numbers 8 and 10. 10 is set to None (default)
 
Please let me know if you need other information.  I appreciate you taking the time to look at this.
 
BTW – ExplorerPatcher is a really terrific app!

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
www.ClarityAndPerformance.com <http://www.clarityandperformance.com/> 
www.ActFromChoice.com <http://www.actfromchoice.com/> 
510.495.4444
--------------------------------------------------------------------------------------

**Comment by tallman69** [comment]
Have you tried this:  Right click on an open spot on the taskbar and uncheck `Lock the taskbar`, then right click just to the left of the double line separator at the right end of the quick launch toolbar, at the top of the context menu that opens expand `View` and click on `Large Icons`.  Does that work for you?  That is how I change the icon size in my quick launch toolbar.  When I am done changing the icon size I relock the taskbar.

**Comment by tallman69** [comment]
Looking at the image you posted, you already have the taskbar unlocked.

**Comment by robtgmann** [comment]
MY HERO.
 
That did it!
 
Many, many thanks.
 
 

Author: ACT FROM CHOICE 
Read the Kirkus and other reviews   <http://www.actfromchoice.com/sample-reviews-and-press> here
Find it on Amazon and other booksellers      
Please follow me on Twitter: @actfromchoice
And join me on LinkedIn
----------------------------------------------------------------------------------------
Robert Goldmann
Clarity & Performance Coaching
Executives, Professionals, Business owners
www.ClarityAndPerformance.com <http://www.clarityandperformance.com/> 
www.ActFromChoice.com <http://www.actfromchoice.com/> 
510.495.4444
--------------------------------------------------------------------------------------

**Comment by tallman69** [comment]
Glad it worked!  Please mark this question as answered.
